envs/env_base.py

```diff
@@ -21,7 +28,7 @@ def apply_action(self, player, action):
         Apply action as a player.
         Args:
             player: the player
-            action: the action
+            action: the action (tuple)
```

Comment:
the action is dependent of the environment, it may be a tuple, string or even a number. please revert this back.

Reply:
done.